zerver/openapi/zulip.yaml

```diff
@@ -4224,8 +4224,12 @@ paths:
               schema:
                 allOf:
                   - $ref: "#/components/schemas/JsonSuccessBase"
-                  - $ref: "#/components/schemas/SuccessDescription"
                   - additionalProperties: false
+                    description: |
+                      When a request is successful, this endpoint returns a dictionary
+                      containing the following (in addition to the `msg` and `result` keys
+                      present in all Zulip API responses).
```

Comment:
Question: Do we need this paragraph?  I would sorta expect that the `msg` and `result` keys would appear in the documentation these days, and it is just legacy content that we can delete.  (It certainly doesn't seem to add any extra detail).

Reply:
They do clearly appear in examples, but not in parameters from what I can see, and they are being mentioned here instead.